docs/scan.md

```diff
@@ -0,0 +1,86 @@
+
+#### Notable Fields
+
+You'll find a **`results`** object, which contains a list of file paths corresponding to detected token data; for example:
+
+```json
+  "results": {
+    "detect_secrets/plugins/private_key.py": [
+      {
+        "hashed_secret": "513e0a36963ae1e8431c041b744679ee578b7c44",
+        "is_secret": false,
+        "is_verified": false,
+        "line_number": 45,
+        "type": "Private Key",
+        "verified_result": null
+      },
+```
+
+| Field           | Description                                                                                                                                                                                                                 |
+| --------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `hashed_secret` | The hash of the detected secret. The baseline file will not record raw secrets. To see them in plaintext, run `detect-secrets audit --display-results .secrets.baseline`.                                                   |
+| `is_secret`     | This field is manually set when interactively auditing a baseline file (`detect-secrets audit .secrets.baseline`). This field is for auditing / record-keeping purposes.results                                                    |
+| `is_verified`   | Set automatically based off active secret validation; however, there are [those which are _not_ validated](#secret-verification). If this field is true, it means the associated token is active, and requires remediation. |
+| `line_number`   | The line number that the secret is found on.                                                                                                                                                                                |
+| `type`          | The secret type.                                                                                                                                                                                                            |
+
+## What Gets Scanned?
+
+The repository's files are scanned in their current state. `detect-secrets` will not run a "deep scan" of the repository (i.e. full commit history).
+
+### Secret Verification
+
+Not only does scanning identify potential tokens, it also verifies if certain types of tokens are active ([verifiable tokens list](./developer-tool-faq.md#what-kinds-of-tokens-does-detect-secrets-find)). If any `is_verified` fields in your baseline are set to `true`, be sure to remediate the associated tokens and re-run the scan.
+
+## How It’s Used
```

Comment:
What do you think about moving this section higher up towards the beginning? I think it would be better to have this section in between `How It Works` and `Pre-commit Hook`

Reply:
I agree that `How It's Used` fits better directly below `How It Works`. I've moved it up!